modules/core/misc/objc/gen_dict.json

```diff
@@ -167,7 +167,9 @@
             "from_cpp": "[Point3i fromNative:%(n)s]"
         },
         "Range": {
-            "objc_type": "Range*"
+            "objc_type": "Range*",
+            "to_cpp": "%(n)s.nativeRef",
```

Comment:
Looks good - let me know if that works OK - if it doesn't then I'll take a closer look

Reply:
OK - let me take a closer look